pkg/document/json/json.go

```diff
@@ -37,3 +43,80 @@ func toOriginal(elem crdt.Element) crdt.Element {
 
 	panic("unsupported type")
 }
+
+func buildCRDTElement(
+	context *change.Context,
+	value interface{},
+	ticket *time.Ticket,
+) crdt.Element {
```

Comment:
I followed the setInternal function interface.

Because Set operation pass the function value(creator) to setInternal function. The function value contain element creation code. 
 
https://github.com/yorkie-team/yorkie/blob/3876b8f4f6ba171ea365ec77514b00fe99b71ec3/pkg/document/json/object.go#L106-L116

https://github.com/yorkie-team/yorkie/blob/3876b8f4f6ba171ea365ec77514b00fe99b71ec3/pkg/document/json/object.go#L327-L352

To keep the previous context, i made the buildCRDTEelement funciton to compatitable the setInternal function. 
Becuase edited SetNewObject use the buildCRDTElement as a return value of function value (creator).

```go
case 2: // key with value
k := kv[0].(string)
v := p.setInternal(k, func(ticket *time.Ticket) crdt.Element {
	json := kv[1].(map[string]interface{})
	return buildCRDTElement(p.context, json, ticket)
})
```

Therefore, there is room for the creator (function value) of all Set functions in "json/object.go" to be refactored by buildCRDTElement.

Reply:
fixed 

https://github.com/yorkie-team/yorkie/blob/a7e7c90d76ac255d255ef18e97084bfd70bfec59/pkg/document/json/json.go#L80-L119